app/views/users/show.html.erb

```diff
@@ -1 +1,15 @@
-<h1>List of user by given id</h1>
\ No newline at end of file
+<div class="user-page">
+  <%= render "shared/details", user: @user %>
+    <div class="user-bio">
+     <strong class="bio">Bio: </strong> <br>
+     <p><%= @user.bio %></p>
+    </div>
+    <div class="user-posts">
+    <% @user.posts.each do |post| %>
```

Comment:
- [ ]  Kindly make use of the `most_recent_posts` callback function you created in your user model to render the 3 most recent posts for a user.